sagemaker-triton/TensorFlow/Deploy-TensorFlow-Model-Using-NVIDIA-Triton.ipynb

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
<ol><li><a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3543/#Transform-TensorFlow-Model-structure" target="_blank">Transform TensorFlow Model structure</a></li><li>  1. <a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3543/#Inspect-the-model-using-the-saved_model_cli" target="_blank">Inspect the model using the saved_model_cli</a> &nbsp;  1. <a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3543/#Create-the-config.pbtxt" target="_blank">Create the config.pbtxt</a> &nbsp;  1. <a href="https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3543/#Create-the-tar-ball-in-the-required-Triton-structure" target="_blank">Create the tar ball in the required Triton structure</a> </li></ol><p>^ This above is odd that there are 3 1. all in the same row, should be cleaned up. Perhaps make it indented with a., b., c.? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
@rdamazon Markdown doesn't always render properly on ReviewNB - by using the `1.` numbering for top-level and sub-headings, GitHub renders it into `1,2,3` and `A,B,C`.

Reply:
<p>For all links, please use the page title as link text, ie <code>[example]</code> -&gt; <code>[Deploy a Trained TensorFlow V2 Model example]</code>  </p>

Reply:
Done

Reply:
Done